midend/interpreter.cpp

```diff
@@ -1095,11 +1110,25 @@ void ExpressionEvaluator::postorder(const IR::MethodCallExpression* expression)
         auto bim = mi->to<BuiltInMethod>();
         auto base = get(bim->appliedTo);
         cstring name = bim->name.name;
+        const IR::Expression* node = nullptr;
```

Comment:
Please add a comment that this code is necessary to find the parent and to check whether the parent is a header union.